_constitution/sse_2018_constitution.md

```diff
@@ -5,95 +5,128 @@ title: SSE Constitution
+2. Members should attend events sponsored / held by the Organization.
 3. Members should offer suggestions for and help plan activities run by the Organization.
 4. Members should seek opportunities to improve the Software Engineering program and community.
 5. Members should encourage other interested students to participate in events held by the Organization.
+6. Members should attempt to keep pace with trends and current affairs related to technology.
 
-Members should attempt to keep pace with trends and current affairs related to technology.
 ### Section 3: Types of Members
-#### 1. REGULAR members
-REGULAR members are those members who attend every Monthly meeting and have the right to vote in any non-executive organization vote. Each member will get ONE(1) unexcused absence from monthly meetings before becoming an INACTIVE member. Given extenuating circumstances, a regular member may request to be excused by any member of the Executive board.
-#### 2. INACTIVE members
-INACTIVE members are those members who have not met the requirements to be a regular member or have been demoted by a simple majority of the executive board. INACTIVE members do not get to participate in any executive or non-executive organization vote. INACTIVE members may be promoted back to a REGULAR member by a simple majority vote by the Executive board
-#### 3. ALUMNI members
-ALUMNI members are those who have graduated from MSOE with REGULAR member standing or those who have been granted ALUMNI status by a simple majority vote of the Executive board. ALUMNI members do not have the right to vote unless all ALUMNI members have been given the right to vote on a specific vote, such permission being granted by a ⅔ majority vote of the Executive board.
-#### 4. SPONSOR members
-SPONSOR membership is granted to any organization, company, or individual (including professors) by a simple majority vote of the Executive board. SPONSOR members do not have the right to vote unless all SPONSOR members have been given the right to vote on a specific vote, such permission being granted by a ⅔ majority vote of the Executive board.
-The Faculty Advisor is automatically a SPONSOR member.
+1. **REGULAR members**
+    * REGULAR members are those members who have earned at least 10 points in the current quarter and have the right to vote in any non-executive organization vote. Regular members may earn points for the following:
+        * General Meeting: 10 pts
+        * Event Attendee: 10 pts
+        * Helping w/ Event/Project: 20 pts
+            * Game development club, website, any event
+        * Leadership Role in Organization/Project: 30 pts
+            * Game development club, website, any event
+    * Members with zero points at any given time are considered INACTIVE.
+    * Points reset to zero at the beginning of each quarter. At the end of each quarter, a raffle will be held and each 10pts had by member corresponds to one raffle entry.
```

Comment:
Space after the "10" in "10pts" to match the other previous pts format in this section